stable/clamav/Chart.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: v1
-appVersion: "1.0"
+appVersion: "1.6"
```

Comment:
why this change here? also the values.yaml is using the tag latest this is not a good practice, can you update to a version tag?

Reply:
my bad, it's fixed now. thanks!